docs/providers/aws/guide/serverless.yml.md

```diff
@@ -238,11 +238,12 @@ provider:
   # Endpoint type for API Gateway REST API: edge or regional (default: edge)
   endpointType: regional
   # Use a custom name for the websockets API
-  websocketsApiName: custom-websockets-api-name
-  # custom route selection expression
-  websocketsApiRouteSelectionExpression: $request.body.route
-  # Use a custom description for the websockets API
-  websocketsDescription: Custom Serverless Websockets
+  websockets:
+    apiName: custom-websockets-api-name
+    # custom route selection expression
+    apiRouteSelectionExpression: $request.body.route
+    # Use a custom description for the websockets API
+    description: Custom Serverless Websockets
```

Comment:
Could you move `websockets` below `apiGateway`? (I think `apiGateway` is used by more people, we should make it a bit more visible)

Also there's the `websocketApiId` property that isn't moved to `websockets`, is that intentional?

Reply:
Matthieu, I will move Websockets after APIGateway. I didn't group websocketApiID with Websockets because it was already a property of APIGateway?